### PR TITLE
docs: add CoC

### PR DESCRIPTION
**Pull request by sinedied**
Add a CoC based on the one from Angular project, which is most simple & effective one I could find :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Contribuer au projet
+
+You can contribute to this project by creating a [Pull Request](https://github.com/Ldoppea/first-steps-dev/pulls). Contributors must abide to this project [Code of Conduct](CODE_OF_CONDUCT.md)
+
+For example, you can propose a new link, new category, delete a linked that became invalid, fix typos or tags, or propose new ideas.
```
Comment (Ldoppea) [inline CONTRIBUTING.md]: Small typo here : should be "delete a **link** that became [...]"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Be sure to have a look to this checklist before submitting a new PR:
+
+- [ ] The proposed link should target content preferably in the same language as the list it's added to. You can however propose English content to non-english lists if it's high quality and add the `[🇺🇸]` suffix after the link.
+- [ ] The linked content is free and non-gated (i.e. it should not require a subscription to access it).
+- [ ] The linked content does not contain any offensive material of any sort that would result in a breach of the [Code of Conduct](CODE_OF_CONDUCT.md).
+- [ ] Add one or more of these tags after the link to give more information about the content. If more that one tag apply, put them in order of relevance, starting with the best one matching.
+   * `What`: the link presents the concept, tool or methodology and explains its foundations
+   * `Why`: the link explains in what cases a developer may need to use this concept, tool or methodology
+   * `How`: the link explains the basics necessary to set up this concept, this tool or this methodology
+   * `Resources`: the link groups together resources to learn about this concept, tool or methodology
```
Comment (Ldoppea) [inline CONTRIBUTING.md:17]: this section is wrongly merged

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+You can contribute to this project by creating a [Pull Request](https://github.com/Ldoppea/first-steps-dev/pulls). Contributors must abide to this project [Code of Conduct](CODE_OF_CONDUCT.md)
+
+For example, you can propose a new link, new category, delete a linked that became invalid, fix typos or tags, or propose new ideas.
+
+Be sure to have a look to this checklist before submitting a new PR:
+
+- [ ] The proposed link should target content preferably in the same language as the list it's added to. You can however propose English content to non-english lists if it's high quality and add the `[🇺🇸]` suffix after the link.
+- [ ] The linked content is free and non-gated (i.e. it should not require a subscription to access it).
+- [ ] The linked content does not contain any offensive material of any sort that would result in a breach of the [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (Ldoppea) [inline CONTRIBUTING.md:11]: Maybe we can add something like:
- The linked content is added to an existing category or a new created one. If you are not sure of the category you may create your PR as a draft and start a discussion about it.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sinedied) [reply CONTRIBUTING.md:17]: This is where I intended it to be, as a reminded for each tag 😄

Comment (Ldoppea) [reply CONTRIBUTING.md:17]: Oh, I didn't notice the list increment, sorry :D

**Pull request merged**